bot/exts/discord/discord.py

```diff
@@ -1,13 +1,20 @@
-from typing import NoReturn
+from typing import NoReturn, Optional
+
 import disnake
-from disnake.ext.commands.errors import BadArgument
+from disnake import Guild, Invite, User
+from disnake.ext import commands
+from disnake.ext.commands.errors import (
+    BadArgument,
+    BadInviteArgument,
+    ConversionError,
+    GuildNotFound,
+)
```

Comment:
This is pretty long. What do you think about just using `commands.BadArgument` etc. to get the errors? Either that or `from disnake.ext.commands import errors` and then `errors.BadArgument` etc.?